invoke-ai/InvokeAI
# Strip whitespace from model URLs

## What type of PR is this? (check all applicable)



## Have you discussed this change with the InvokeAI team?

      
## Have you updated all relevant documentation?


## Description

This PR strips leading and trailing whitespace from URLs that are entered into either the Web Model Manager import field, or using the TUI.

## Related Tickets & Documents

Closes #4536


## QA Instructions, Screenshots, Recordings

Try to import a URL with leading or trailing whitespace. Should not work in current main. This PR should fix it.
